mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown

```diff
@@ -198,3 +198,9 @@ Users will need to check their configuration for any `google_vpc_access_connecto
 resource blocks that contain both fields in a conflicting pair, and remove one of those fields.
 The fields that are removed from the configuration will still have Computed values,
 that are derived from the API.
+
+## Resource: `google_service_networking_connection`
+
+### Field `update_on_creation_fail` enforces an update in case of creation failure
```

Comment:
This isn't a breaking change anymore right? Could this just go into 5.x?

Reply:
Yeah, it can go to 5.x

Reply:
do we want to do that?

Reply:
I will move it to the main branch. I will close this PR and open a new PR.